moveit_ros/moveit_servo/demos/cpp_interface/demo_pose.cpp

```diff
@@ -51,6 +51,12 @@
 using moveit::getLogger;
 using namespace moveit_servo;
 
+namespace
+{
+constexpr auto K_BASE_FRAME = "panda_link0";
+constexpr auto K_TIP_FRAME = "panda_link8";
```

Comment:
Not in this PR, but we should move this demo to the tutorials. What do you think?

Reply:
I think we made the conscious decision to keep a few simple demos in the core repo itself during development of this project, so I'd rather keep as is until/unless we decide to do something sweeping with the whole repo and the way we do demos.

Reply:
I'm with sea-bass here

Reply:
That's fine with me :+1: